Slim/Menu/BrowseLibrary.pm

```diff
@@ -1525,7 +1525,7 @@ sub _albums {
 				}
 				else {
 					$_->{'artists'}    = [ $_->{'artist'} ];
-					$_->{'artist_ids'} = [ $_->{'id'} ];
+					$_->{'artist_ids'} = [ $_->{'artist_id'} ];
```

Comment:
An unrelated bug: $_{'id'} is the album_id!

Reply:
Wow! Good catch. 10 years, and nobody hit that code?!?

Reply:
I think it's more or less impossible for `artist_ids` not to be already populated.